app/Http/Authentication/AmazonLoginAuthenticator.php

```diff
@@ -81,7 +81,7 @@ private function verifyUserTokenMatchesAmazonToken(string $accessToken) : bool
         return true;
     }
 
-    private function exchangeAccessTokenForDecodedUserProfile(string $accessToken)
+    private function exchangeAccessTokenForDecodedUserProfile(string $accessToken): \stdClass
```

Comment:
This is new to me!  Just read up on it some!  I like that there is a return type for this now.  I had trouble figuring out what to do for mixed return types. 🎉